CLAUDE.md

```diff
@@ -0,0 +1,22 @@
+# Commands for ElizaOS Contributor Analytics
+
+## Build & Development
+- `bun run dev` - Start development server
+- `bun run build` - Build production site
+- `bun run check` - Run linter and type checks
+- `bun run lint` - Run ESLint only
+- `bunx tsc --noEmit` - Run TypeScript checks
+- `bun run serve` - Serve built site
+- `bun run init-db` - Initialize database
```

Comment:
- errors on bun run build and bun run check (schemas defined but never used, unexpected any, etc)
- `error: Script not found "init-db"`
- errors with `bun run lint`